package.json

```diff
@@ -6,14 +6,14 @@
     "start": "npm run dev",
     "dev": "vite",
     "build": "run-p type-check build-only",
+    "build-only": "vite build",
     "preview": "vite preview",
     "test": "npm run test:unit",
     "test:unit": "vitest --environment jsdom --root .",
     "test:unit:ci": "vitest run --environment jsdom --coverage",
-    "test:e2e": "start-server-and-test 'vite preview' :4173 'cypress run --e2e'",
+    "test:e2e": "start-server-and-test preview :4173 'cypress run --e2e'",
```

Comment:
Wondering if we still need this. When running it locally after `npm run build-only -- --mode=e2e` the test looking for the lux logo in `assets/header` breaks on my side btw. 

Might also be helpful to document this a bit, to explain that `npm run build` => `npm run test:e2e` doesn't make sense to run for example.

Reply:
Thanks for reporting this. I have update the test looking for the logo: now it checks if it loads the image (it tests the image natural width) instead of checking the src content. This is because in production, base64 source for image are used instead of uri.

I have added some documentation in the README.md about test commands.